application/classes/Ushahidi/Repository/Form/Attribute.php

```diff
@@ -234,6 +234,19 @@ public function getByForm($form_id)
 		return $this->getCollection($results->as_array());
 	}
 
+	public function getByForms($form_ids) {
```

Comment:
TODO: I should move the attributes query in postexport repo to live here and call this repo

Reply:
done